core/arch/arm/kernel/secure_partition.c

```diff
@@ -525,6 +526,87 @@ static TEE_Result handle_fdt_dev_regions(struct sp_ctx *ctx, void *fdt)
 	return TEE_SUCCESS;
 }
 
+#ifdef CFG_CORE_TPM_EVENT_LOG
```

Comment:
I looks like this ifdef could be removed and leave it to the compile to optimize it away. That we we still have this function compile tested even without CFG_CORE_TPM_EVENT_LOG.

Reply:
Sounds reasonable, done.